src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -140,6 +147,10 @@ public Operation getOperation() {
     return Operation.valueOf(operation.toUpperCase());
   }
 
+  public WriteDisposition getWriteDisposition() {
+    return WriteDisposition.valueOf(writeDisposition.toUpperCase());
```

Comment:
with the change to use a boolean, this just returns truncate if true, append if false or null.

Reply:
Fixed!